preferences/src/main/java/com/itsaky/androidide/preferences/internal/editorPreferences.kt

```diff
@@ -114,10 +116,22 @@ var wordwrap: Boolean
     prefManager.putBoolean(WORD_WRAP, value)
   }
 
-var showFileTreeButton: Boolean
-  get() = prefManager.getBoolean(SHOW_FILE_TREE_BUTTON, false)
-  set(value) {
-    prefManager.putBoolean(SHOW_FILE_TREE_BUTTON, value)
+/** By default File Tree button is hidden into the overflow menu on devices running
+ * Android 11 and older and on those, where gesture navigation is disabled at the moment of
+ * first launching the app.
+ *
+ * Getting the navigation mode, however, requires a context.
+ */
+var hideFileTreeButton: Boolean
+  get() = prefManager.getBoolean(
+    HIDE_FILE_TREE_BUTTON,
+    // TODO: Decide whether should we consider the fact that Gesture Navigation is enabled to set a
```

Comment:
Address the following TODO :eyes:

Reply:
> Decide whether should we consider the fact that Gesture Navigation is enabled

Yeah, sure.

> Getting the navigation mode requires a Context

You could use `BaseApplication.getBaseInstance()` to get the application context.